build/calendar/calendar-debug.js

```diff
@@ -126,6 +142,10 @@ Y.Calendar = Y.extend(Calendar, Y.CalendarBase, {
         var allHilitedNodes = this.get("contentBox").all("." + CAL_DAY_HILITED);
         if (allHilitedNodes) {
             allHilitedNodes.removeClass(CAL_DAY_HILITED);
+
+            for (let i = 0; i < allHilitedNodes.length; i++) {
```

Comment:
Since this is an old code base, it's probably best to just use var instead of let for consistency.